api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -80,7 +81,7 @@ private ClusterRequest createFirecloudClusterRequest(
     nbExtensions.put(
         "aou-activity-checker-extension", gcsPrefix + "/activity-checker-extension.js");
 
-    Workspace workspace = workspaceService.getRequired(workspaceNamespace, workspaceName);
+    Workspace workspace = workspaceService.getByName(workspaceNamespace, workspaceName);
```

Comment:
This is an issue I noticed, `getRequired` uses the FC name, but we were passing up the AoU name. This is a piece of tech debt that we definitely should tackle in the near future, because multiple things named almost the same is a pain in the neck.

Reply:
Yikes!  I'm seeing errors locally which might be caused by this, and are driving me nuts.  I'll take a poke at this.

Reply:
Ah - that's not good. We should fix this in the other direction. We want to use the firecloud name here; the rest of the stack should pass the correct value down.

Reply:
Actually nope.  This can of worms is much larger than I'd guessed.

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/RW-3890

Reply:
Agreed we should fix it in the other direction, I just picked the easiest possible fix. Thank you for filing Joel.

Reply:
I'm fixing this line in #2816 anyway